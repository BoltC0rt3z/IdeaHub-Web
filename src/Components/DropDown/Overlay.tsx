import React from 'react';
import { Dropdown } from '@primer/components';

import styles from './Overlay.module.scss';

interface Props {
  className?: string;
}

const Overlay: React.FC<Props> = (props) => {
  const { className } = props;
  const classNames: string = [styles.overlayContainer, className].join(' ');

  return (
    <div className={classNames}>
      <Dropdown.Menu>
        <Dropdown.Item>Item 1</Dropdown.Item>
      </Dropdown.Menu>
    </div>
  );
};

export default Overlay;
