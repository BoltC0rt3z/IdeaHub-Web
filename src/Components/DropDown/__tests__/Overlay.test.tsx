import React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import Overlay from '../Overlay';

describe('Tests for overlay', () => {
  const wrapper: ReactWrapper = mount(<Overlay />);

  it('Should render overlay without crushing', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
