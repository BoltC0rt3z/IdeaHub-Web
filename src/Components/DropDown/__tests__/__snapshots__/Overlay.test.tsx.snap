// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for overlay Should render overlay without crushing 1`] = `ReactWrapper {}`;
